packages/@lightningjs/ui-components/src/components/Base/Base.js

```diff
@@ -54,14 +54,20 @@ class Base extends lng.Component {
   _update() {}
 
   _focus() {
-    if (this.shouldSmooth === undefined) this.shouldSmooth = true;
+    super._focus();
```

Comment:
I'm not sure we need this super call? as far as I can tell `lng.Component._focus()` doesn't exist
https://github.com/rdkcentral/Lightning/blob/master/src/application/Component.d.mts

Reply:
https://github.com/rdkcentral/Lightning/blob/4a3ecc189a7a978790346143e62cb99ac7f1c30c/src/application/Component.d.mts#L916 Is this not it?